build-system/tasks/extension-helpers.js

```diff
@@ -734,7 +734,7 @@ async function buildBentoExtensionJs(dir, name, options) {
   );
   await buildExtensionJs(dir, name, {
     ...options,
-    externalDependencies: Object.values(remapDependencies),
+    externalDependencies: [...new Set(Object.values(remapDependencies))],
```

Comment:
I guess we want to make sure the `remapDependencies` has unique values?

Reply:
yes, right now this array is `['../bento.mjs', '../bento.mjs', '../bento.mjs', '../bento.mjs', '../bento.mjs']` for now reason